src/qt/addresstablemodel.h

```diff
@@ -12,6 +12,7 @@ enum class OutputType;
 
 class AddressTablePriv;
 class WalletModel;
+class CAddressBookData;
```

Comment:
forward declaration unused. Please remove

Reply:
Good catch, thanks. Will spend some time figuring out a sensible `clang-tidy` config to catch this sort of stuff.